apps/coscrad-frontend/src/store/slices/notes/note-slice.ts

```diff
@@ -1,12 +1,12 @@
-import { INoteViewModel } from '@coscrad/api-interfaces';
+import { INoteViewModel, WithTags } from '@coscrad/api-interfaces';
 import { createSlice } from '@reduxjs/toolkit';
 import { buildInitialLoadableState } from '../utils';
 import { buildReducersForThunk } from '../utils/build-reducers-for-thunk';
 import { NOTES } from './constants';
 import { fetchNotes } from './thunks';
 import { NoteSliceState } from './types';
 
-const initialState: NoteSliceState = buildInitialLoadableState<INoteViewModel[]>();
+const initialState: NoteSliceState = buildInitialLoadableState<WithTags<INoteViewModel>[]>();
```

Comment:
It would be nice if we didn’t have to wrap this in everywhere. The API response mixes in the tags. So we want to constrain the backend view-model class without tags but the API response with tags. As far as the front-end is concerned, the tags are always there. One can question why we expose the view models to the frontend without tags.

Reply:
This has been fixed.